lib/ActionField.js

```diff
@@ -30,7 +30,7 @@ class ActionField extends Immutable.Record({
 			map.set('name', json.name || map.name);
 			map.set('title', json.title || map.title);
 			map.set('type', json.type || map.type);
-			map.set('value', json.value || map.value);
+			map.set('value', (json.value === false || json.value === 0) ? json.value : json.value || map.value);
```

Comment:
What does "stopping" and "getting through" mean?  And I'm not 100% sure what the "null" part getting through because of map.value means.  Do you mean that since it defaults to null anyways, effectively it winds up with null?

The point of this code is effectively that if we're provided a value, use it (including null), if we aren't, then use null.

Personally I think the correct behavior here would be to set the value to null whenever the incoming JSON is undefined, else use the value from JSON.